src/SimpleSoftwareIO/SMS/Drivers/InfobipSMS.php

```diff
@@ -0,0 +1,74 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: m
+ * Date: 4/3/16
+ * Time: 1:01 PM
+ */
+
+namespace SimpleSoftwareIO\SMS\Drivers;
+
+use infobip\models\SMSRequest;
+use infobip\SmsClient;
+use SimpleSoftwareIO\SMS\OutgoingMessage;
+
+/**
+ * Class InfobipSMS
+ * @package SimpleSoftwareIO\SMS\Drivers
+ */
+class InfobipSMS extends AbstractSMS implements DriverInterface
+{
+
+    protected $smsRequest;
+    protected $smsClient;
+
+    /**
+     * @param SmsClient $client
+     */
+    public function __construct(SmsClient $client)
+    {
+        $this->smsClient =  $client;
+    }
+
+
+    protected function processReceive($rawMessage)
+    {
+        // TODO: Implement processReceive() method.
```

Comment:
Any function that is not supported should `throw new \RuntimeException('Infobip does not support Inbound API Calls.');`